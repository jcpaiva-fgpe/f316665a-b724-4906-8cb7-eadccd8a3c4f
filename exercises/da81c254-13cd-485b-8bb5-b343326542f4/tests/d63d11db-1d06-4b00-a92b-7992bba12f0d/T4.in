16 15
18 57
