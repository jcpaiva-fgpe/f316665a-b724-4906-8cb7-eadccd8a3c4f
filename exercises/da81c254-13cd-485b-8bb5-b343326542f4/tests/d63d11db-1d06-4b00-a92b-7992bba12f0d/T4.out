Passaram apenas 162 minutos!
Queres dizer, 2 horas e 42 minutos?!
